docs/sequence-diagrams/api-integration.puml

```diff
@@ -9,9 +9,10 @@ participant "Veraison\nEndorsements\nProvisioning" as V_EndProv order 40
 database "Veraison\nEndorsement Store" as V_EndStore order 50
 
 group keys provisioning and Node ID assignment
+
 	Node -> BE: POST /node/pem, Body: {nodeID, AK_pub, EK_pub}
 	BE -> BE: generate nodeID
-	BE -> BE_DB: store nodeID
+	BE -> BE_DB: store nodeID AK_pub and EK_pub
 	BE -> BE: repackage {nodeID, AK_pub} as CoRIM
 	BE -> V_EndProv: POST /submit, Body: CoRIM{nodeID, AK_pub}
```

Comment:
What does AVK stand for?

Reply:
it's the "attester verification key" CoRIM triple (see [this](https://github.com/veraison/enact-demo/blob/aebdf44e22df952a829307a3c036e6e8bc4414ae/examples/corim-repackage/repackage.go#L31) snippet)

Reply:
I see. I only wished we could have a shorter description here.

Reply:
Yeah, that C-T is quite a mouthful :-) I don't think we should drop it because it's essential to the API.  What we could do (typographically) is adding a line break, e.g.:
```
BE -> V_EndProv: POST /submit,\nC-T: application/corim+cbor; profile=enacttrust-tpm,\nBody: CoRIM{AVK(nodeID, AK_pub)}
```